src/components/Card/Card.tsx

```diff
@@ -11,18 +12,41 @@ function Card({
   flip: boolean;
   onClick: () => void;
 }) {
+  const ref = React.useRef<HTMLDivElement>(document.createElement("div"));
+
+  React.useEffect(() => {
+    let entryPoint = [0, 0];
+    const track = (moveEvent: MouseEvent) => {
+      moveEvent.stopPropagation();
+      ref.current.style.transform = `translate(${
+        (moveEvent.offsetX - entryPoint[0]) * 0.02
+      }px, ${(moveEvent.offsetY - entryPoint[1]) * 0.02}px)`;
```

Comment:
It's hard to tell what's `entryPoint[0]` and `entryPoint[1]`. What if you use object for this?

```ts
let entryPoint = { offsetX: 0, offsetY : 0 }
...
(event.offsetY - entryPoint.offsetY) * 0.02
```